scripts/pre_commit_linter.py

```diff
@@ -1312,6 +1334,82 @@ def _check_js_component_name_and_count(self):
             print ''
             return summary_messages
 
+    def _check_mandatory_patterns(self):
+        """This function checks that all JS files contain the mandatory
+        patternss.
+        """
+        if self.verbose_mode_enabled:
+            print 'Starting mandatory patterns check'
+            print '----------------------------------------'
+
+        summary_messages = []
+        failed = False
+        all_files_to_check = [
+            filepath for filepath in self.all_filepaths if not any(
+                fnmatch.fnmatch(filepath, pattern) for pattern in
+                EXCLUDED_PATHS + GENERATED_FILE_PATHS +
```

Comment:
If we're excluding these files, we should list them out in under `excluded_files` in `MANDATORY_PATTERNS_*` list.

Reply:
Done!